.github/workflows/format-code.yml

```diff
@@ -0,0 +1,54 @@
+name: "Format code"
+
+on:
+  issue_comment:
+    types: [created]
+
+jobs:
```

Comment:
I think we need an always run job, like the rebase action.

https://github.community/t5/GitHub-Actions/Workflow-is-failing-if-no-job-can-be-ran-due-to-condition/m-p/38186#M3250

Reply:
Good catch. Done